scripts/update-versions.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+versioned_packages="\(plutus-core\|plutus-ledger-api\|plutus-tx\|plutus-tx-plugin\)"
```

Comment:
what about `word-array` and `prettyprinter-configurable`? They're dependencies of those packages, so if they change we need to release updated versions, which means we need to version them somehow. I think the lockstep consideration means we version them the same as everything else.

Reply:
maybe "versioned_packages_regex" since that's what it is. You could have a bash array of the packages and then turn it into the regex, slightly easier to update in future

Reply:
I guess we could special case `word-array` and `prettyprinter-configurable` since they change so infrequently :thinking:

Reply:
Special casing them in the sense that we only release them if they are changed sounds good to me.

Reply:
In that case how about making `update-versions` take the list of packages to update from its arguments, using the current list as a default if it's not supplied. That way if we do want to bump `word-array` we can just run `update-versions.sh word-array`.

Reply:
Yup, that's exactly what I'm doing.

Reply:
See [04f13fe](https://github.com/input-output-hk/plutus/pull/4916/commits/04f13fe3e08ccc2d94f98631bcfa6ac378b1dd42)